index.html

```diff
@@ -2054,7 +2054,7 @@ <h3>The <dfn>subscribeEvent()</dfn> method</h3>
             with a {{TypeError}} and stop.
           </li>
           <li>
-            If |thing|.{{ConsumedThing/[[activeSubscriptions]]}}[|eventName|] does not [=map/exist=], [=reject=] |promise| with a {{NotAllowedError}} and stop.
+            If |thing|.{{ConsumedThing/[[activeSubscriptions]]}}[|eventName|] does [=map/exist=], [=reject=] |promise| with a {{NotAllowedError}} and stop.
```

Comment:
Tiny nit, otherwise: LGTM :)

```suggestion
            If |thing|.{{ConsumedThing/[[activeSubscriptions]]}}[|eventName|] [=map/exists=], [=reject=] |promise| with a {{NotAllowedError}} and stop.
```